src/MQClientAPIImpl.cpp

```diff
@@ -430,10 +430,6 @@ void MQClientAPIImpl::sendMessageAsync(const string& addr,
   }
 }
 
-void MQClientAPIImpl::deleteOpaqueForDropPullRequest(const MQMessageQueue& mq, int opaque) {
```

Comment:
delete getLatestPullRequestOpaque and setLatestPullRequestOpaque the same time in this pr ?

Reply:
Yes, I have removed them.